packages/kernel-5.15/config-bottlerocket

```diff
@@ -0,0 +1,136 @@
+# Because Bottlerocket does not have an initramfs, modules required to mount
+# the root filesystem must be set to y.
```

Comment:
Please keep "rebasing" this on 5.10 kernel config changes until this merges, and then please nag others to keep this config current.

Reply:
There are two new classes that need to be added:

```
[    0.956969] SELinux:  Class mctp_socket not defined in policy.
[    0.957602] SELinux:  Class anon_inode not defined in policy.
[    0.958165] SELinux: the above unknown classes and permissions will be denied
[    0.960123] SELinux:  policy capability network_peer_controls=1
[    0.960731] SELinux:  policy capability open_perms=1
[    0.961217] SELinux:  policy capability extended_socket_class=1
[    0.961812] SELinux:  policy capability always_check_network=0
[    0.962377] SELinux:  policy capability cgroup_seclabel=1
[    0.962907] SELinux:  policy capability nnp_nosuid_transition=1
[    0.963484] SELinux:  policy capability genfs_seclabel_symlinks=1
[    0.964114] SELinux:  policy capability ioctl_skip_cloexec=0
```